src/main/utils/utils.ts

```diff
@@ -53,7 +55,7 @@ export const convertISODateTimeToUTCFormat = (date: string, trimSeconds?: boolea
   if (result === 'Invalid Date') {
     return '';
   } else if (trimSeconds) {
-    return result.substring(0, result.length-12);
+    return format(new Date(date), 'EEE, dd MMM yyyy HH:mm') + ' GMT';
```

Comment:
Looks good - just wonder if the utility method name should make it clear that it's a "short" datetime, or specific to login time?

Reply:
I couldn't think of a way of doing it without code duplication, but I guess I can